tests/test_wfirst.py

```diff
@@ -536,15 +536,24 @@ def test_wfirst_detectors():
     im_list = [im0_1,im0_2]*4
 
     im_1.applyPersistence(im_list,galsim.wfirst.persistence_coefficients)
-    galsim.wfirst.applyPersistence(im_2, im_list)
+    galsim.wfirst.applyPersistence(im_2, im_list, method='linear') #check the linear method
     assert im_2.scale == im_1.scale
     assert im_2.wcs == im_1.wcs
     assert im_2.dtype == im_1.dtype
     assert im_2.bounds == im_1.bounds
     np.testing.assert_array_equal(
-        im_2.array, im_1.array,
-        err_msg='Persistence results depend on function used.')
+       im_2.array, im_1.array,
+       err_msg='Persistence results depend on function used.')
+
+    im_unit = galsim.Image(np.ones((2,2)), copy=True)
+    im_f = im_unit*0.0
+    im_f_list = [im_unit*1.E3, im_unit*1.E5]
+    galsim.wfirst.applyPersistence(im_f, im_f_list, method='fermi') #check fermi method
+    #Check the functionality of the fermi method by comparing with pre-calculated values.
+    assert np.allclose( im_f.array, np.ones((2,2))*8.30515, rtol=1.E-06 ), 'Error in Fermi persistence model'
+
     assert_raises(TypeError, galsim.wfirst.applyPersistence, im_2, im0)
+    assert_raises(galsim.GalSimValueError, galsim.wfirst.applyPersistence, im_2, im_list, method='wrong method')
```

Comment:
Also need a test that running `applyPersistence` with no specified `method` is the same as `'fermi'`, not `'linear'`.

Reply:
A test function is added.